test_email_validation.py

```diff
@@ -0,0 +1,20 @@
+
+import re
+import unittest
+
+
+class EmailValidationTest(unittest.TestCase):
+    """unit test for EmailValidation"""
+    def test_validate_email(self):
+        self.assertTrue(e_mail, validate_email())
+
+
+def validate_email(a):
```

Comment:
1) not here
2) argument name sucks
3) should be returning value
4) should have a comprehensive docstring